ext/hal/nordic/nrfx_config_nrf52840.h

```diff
@@ -1826,16 +1826,16 @@
 
 // </e>
 
-// <q> NRFX_SPIM3_NRF52840_ANOMALY_198_WORKAROUND_ENABLED  - Enables nRF52840 anomaly 198 workaround for SPIM3.
- 
-
-// <i> See more in the Errata document located at 
-// <i> https://infocenter.nordicsemi.com/
-
-#ifndef NRFX_SPIM3_NRF52840_ANOMALY_198_WORKAROUND_ENABLED
-#define NRFX_SPIM3_NRF52840_ANOMALY_198_WORKAROUND_ENABLED 0
-#endif
-
+    // <q> NRFX_SPIM3_NRF52840_ANOMALY_198_WORKAROUND_ENABLED  - Enables nRF52840 anomaly 198 workaround for SPIM3.
```

Comment:
did you mean to add this whitespace?

If so, I'd put it in a separate commit

Reply:
I'll fix it